Simplenote/SPAppDelegate.m

```diff
@@ -41,6 +41,7 @@
 @interface SPAppDelegate ()
 
 @property (weak,   nonatomic) SPModalActivityIndicator      *signOutActivityIndicator;
+@property (nonatomic) NSArray *selectedNotesEnteringBackground;
```

Comment:
@charliescheer do you think we could encapsulate this in the NoteList itself?

(So that the AppDelegate doesn't keep track of IndexPath(s))

Reply:
Ooff that would have been so much easier if the whole thing was in swift.  Yes I have made that change. 👍